features/guest_can_login_to_the_service.feature

```diff
@@ -0,0 +1,22 @@
+@javascript
+Feature: Guest can login to the service
+    As a Guest,
+    In order to support the campaigns,
+    I would like to login to the service
+
+    PT story: https://www.pivotaltracker.com/story/show/160308818
+
+    Background:
+        Given the following user exist
+        |email           |password   |
+        |thomas@venue.se |password   |
```

Comment:
be consistent with your table formatting. This is how it looks in othe feature files
```
    Background:
        Given the following users exist
        | email          | role   |
        | user@fan.se    | fan    |
        | user@artist.se | artist |
```